src/ObslugaZamowien3.java

```diff
@@ -0,0 +1,20 @@
+
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+
+
+public class ObslugaZamowien3 {
+    public static void main(String[] args) {
+
+        ArrayList<Orders> listOfOrdersFinal = (ArrayList<Orders>) FileLoader.loadFile("file.csv");
```

Comment:
Please use interface on left side "List" also cast is is not needed possibly wrong?